website/docs/workloads/HPLinpack/HPLinpack-profiles.md

```diff
@@ -0,0 +1,40 @@
+﻿# HPLinpack Workload Profiles
+The following profiles run customer-representative or benchmarking scenarios using the HPLinpack workload.  
+
+* [Workload Details](./HPLinpack.md)  
+
+## PERF-CPU-HPLINPACK.json
+HPL (High Performance Linpack) is a software package that solves a (random) dense linear system in double precision (64 bits) arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available implementation of the High Performance Computing Linpack Benchmark.
+
+The HPL package provides a testing and timing program to quantify the accuracy of the obtained solution as well as the time it took to compute it. The best performance achievable by this software on your system depends on a large variety of factors.
+
+This profile is designed to identify general/broad regressions when compared against a baseline by testing and timing programs that provide complete solutions for the most common problems for linear system of equations.
+
+* [Workload Profile](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-HPLINPACK.json) 
+
+* **Supported Platform/Architectures**
+  * linux-x64
+  * linux-arm64
+
+* **Supports Disconnected Scenarios**  
+  * No. Internet connection required.
+
+* **Dependencies**  
```

Comment:
The dependency part is for actually explaining the dependencies we put in the profile, like compiler installation, and the ARM package. Please put link, and how/why we need that ARM binaries to run Linpack